src/components/plugins/edit/EditHandler.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import _ from "lodash";
+
+export default class EditHandler {
+  selectedRecords: Array<Number>;
+
+  toggleSelect: Function;
+
+  select: Function;
+
+  unselect: Function;
+
+  clearAllSelected: Function;
+
+  constructor() {
+    this.selectedRecords = [];
+
+    this.toggleSelect = this.toggleSelect.bind(this);
+    this.select = this.select.bind(this);
+    this.unselect = this.unselect.bind(this);
+    this.clearAllSelected = this.clearAllSelected.bind(this);
+  }
+
+  toggleSelect(...indices) {
+    indices.forEach((index) => {
+      if (this.selectedRecords.includes(index)) {
+        this.selectedRecords = this.selectedRecords.filter((record) => record !== index);
+      } else {
+        this.selectedRecords.push(index);
+      }
+    });
+  }
```

Comment:
[Modification Request] This appears to be used by only one location - toggling a checkbox on a row. Since this will only get one record for processing, modify the function to reflect that.